README.md

```diff
@@ -41,3 +41,9 @@ We have support for a number of CI systems for a few reasons:
 # Releases
 SharpPcap is released via nuget
 
+# Versions
+
+* 5.4 is the last release in the 5.x series
+* 6.0 brings a series of API breaking changes including:
+  - Removal of deprecated WinPcap class
+  - Switch to optional arguments for ICaptureDevice
```

Comment:
I would recommend using a Dictionary like arguments similar to https://stackoverflow.com/a/60887063/1438522

Reply:
How do they compare in terms of usage? It would help to see how the dictionary approach could be simpler than optional arguments.

Reply:
There are two options I can think of:
* something like a `Dictionary<PcapOpenParam, object>` where `PcapOpenParam` is an enumeration (similar to WinDivert approach here https://www.reqrypt.org/windivert-doc.html#divert_set_param)
```cs
public enum PcapOpenParam {
  OpenFlags,
  ReadTimeout,
  MonitorMode,
  ...
}
```
* A simple `DeviceConfiguration` class, like the one in https://pcapplusplus.github.io/api-docs/structpcpp_1_1_pcap_live_device_1_1_device_configuration.html
In this case sub-classes of `DeviceConfiguration` may be required (for WinDivert specific options for example).

Reply:
Could we use nullable types to make the list more definite? Optional parameters isn't particularly clean here but I'm still figuring it out.

Reply:
@kayoub5 I've got some changes to collapse the API down that I'm sure you've seen. That should make it easier to test and look at the DeviceConfiguration approach too so don't take those changes as the final approach but as a step that should make it easier to look at both approaches.